.github/workflows/build-ami.yml

```diff
@@ -0,0 +1,91 @@
+name: build-and-publish-ami
+
+on:
+  workflow_dispatch:
+    inputs:
+      regions:
+        description: 'Publication regions'
+        required: true
+        default: 'ap-south-1,eu-west-3,eu-west-2,eu-west-1,ap-northeast-2,ap-northeast-1,sa-east-1,ca-central-1,ap-southeast-1,ap-southeast-2,eu-central-1,us-east-1,us-east-2,us-west-1,us-west-2'
```

Comment:
Do these map to where k8s infra's running?

Reply:
These map to the regions in image-builder: https://github.com/kubernetes-sigs/image-builder/blob/2f188e738f961730645269fe942cfcbb0925db7a/images/capi/packer/ami/packer-windows.json#L161

Reply:
Which are the historical regions we published AMIs to. We can scope this down in the future.